### ISSUE TITLE
The 'Join Event' and 'View Event' Buttons are coinciding with each other and styling isn't proper.

### ISSUE THREAD

**Issue by sreejasankuri** [rolanorebelo/ElderApp#13]
The 'Join Event' and 'View Event' Buttons are coinciding with each other and styling isn't proper. This affects usability and accessibility since the spacing is necessary and proper contrast is also necessary for good usability and accessibility. If the background color is light, the text color should be dark and vice versa.

Actual Behavior:
The 'Join Event' and 'View Event' Buttons are coinciding with each other and background color is too light and text color is light too.

Expected Behavior:
There should be appropriate spacing between the 'Join Event' and 'View Event' Buttons.

Before Fix:
![image](https://github.com/rolanorebelo/ElderApp/assets/101545094/220a66e2-76f2-4e4f-945c-ebede38cd767)

After Fix:
![image](https://github.com/rolanorebelo/ElderApp/assets/101545094/e14b71bf-d72e-4a9a-974a-8c467282687d)

**Comment by sreejasankuri** [comment]
This issue was resolved by adding CSS to the 'actionButton' class.
`  <View style={styles.actionButtonsWrapper}>
              <TouchableOpacity
                style={[styles.actionButton, { backgroundColor: '#2D264B' }]}
                onPress={() => handleJoinEvent(event)}>
                <Text style={styles.actionButtonText}>Join Event</Text>
              </TouchableOpacity>
              <TouchableOpacity
                style={[styles.actionButton, { backgroundColor: '#2D264B' }]}
                onPress={() => handleViewEvent(event)}>
                <Text style={styles.actionButtonText}>View Event</Text>
              </TouchableOpacity>
            </View>`

`actionButton: {
  paddingVertical: 10,
  paddingHorizontal: 20, // Increase horizontal padding for more space
  borderRadius: 8,
  alignItems: 'center',
  justifyContent: 'center',
  marginLeft:10
},
`
